yb_migrate/cmd/analyzeSchema.go

```diff
@@ -681,7 +687,7 @@ func generateTxtReport(Report utils.Report) string {
 	}
 	for i := 0; i < len(Report.Issues); i++ {
 		txtstring += "Error in Object " + Report.Issues[i].ObjectType + ":\n"
```

Comment:
i think colon(:) related edit can be done to this line also.
```
txtstring += "Error in Object: " + Report.Issues[i].ObjectType + "\n"
```

Reply:
This is the first line for each issue, which is why I had not removed it as it looks well-formatted

Reply:
ohk got it